packages/cursorless-vscode-e2e/suite/fixtures/recorded/languages/perl/changeArg.yml

```diff
@@ -0,0 +1,22 @@
+languageId: perl
+command:
+  version: 4
+  spokenForm: change arg
+  action: {name: clearAndSetSelection}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: argumentOrParameter}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: some_funky_func( "and", "three", "args" )
+  selections:
+    - anchor: {line: 0, character: 36}
+      active: {line: 0, character: 36}
+  marks: {}
+finalState:
+  documentContents: some_funky_func(  )
```

Comment:
How come this targets all three args rather than just one?

Reply:
see https://github.com/ganezdragon/tree-sitter-perl/issues/23

Reply:
This is now fixed and working the same as the Python implementation